src/ckeditor.jsx

```diff
@@ -30,9 +30,9 @@ class CKEditor extends React.Component {
     var node = React.findDOMNode(this.refs.ck);
     $(node).text(this.toHtml(content));
 
-    var imageBrowserUrl = window.location.href
-      + '&ckImageBrowser=on'
-      + '&siteUrl=' + encodeURIComponent(this.getSiteUrl());
+    var imageBrowserUrl = window.location.pathname + '/ckImageBrowser'
+      + window.location.search + (window.location.search ? '&' : '?')
```

Comment:
window.pathname does not contain the query string, only the path.
It's possible that ?demo will be present at the, that's why all this.